### PR TITLE
Capstone Project PR 

### PR DESCRIPTION
**Pull request by Uchenna-ibute**
1. Add microverse readme template to the project.
2. Add html-css-js linters to the project.
3. Add a .gitignore file to the project.
4. Add the main page and about page HTML files and CSS files.
5. Add icons and background images to the project.
6. Add js file for the functionality of the featured speakers in the HTML file
7. Loom video link    https://www.loom.com/share/a991239eaa5a4d45937ee1754416bf09

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+- **Mobile and desktop versions of Main page**
+- **Mobile and desktop versions of About page**
+- **Responsive webpages developed with Responsive Web Design principle**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (Kossi-stack) [inline README.md]: - [ ] Kindly fix the link to your live demo. it does not work

File: about.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+          <div class="imgg">
+            <img src="img/paystack-logo-vector.png" alt="" srcset="">
+          </div>
+        </div>
+    </section>
+    
+    <footer class="footer">
+      <img class="footer-image" src="img/woman-fashion-logo-design-vector-44166694.jpg" alt="Logo of UC's Fashion Show">
+      <div class="info">
+        <h4 class="footer-title">Uc's Fashion Show 2023</h4>
```
Comment (Kossi-stack) [inline about.html:118]: - [ ] Kindly fix the space between the image and the paragraph in the footer
Screenshot 2023-01-04 at 20 40 32

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Uchenna-ibute,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.

## Required Changes ♻️
 - [ ] Kindly edit the PR's title to something more descriptive. it can be "Capstone Project PR or HTML Capstone Project ".
 - [ ] Kindly capitalize the first letter of your commit message in the next milestone so it looks professional.
 - [ ] Kindly add a recorded video presenting the features of the project. The video contains a clear and objective description of the project in professional English. The video should contain a highlight about a piece of the code the student finds interesting or is proud of making.
 - [ ] Make sure to add the corresponding path to your MIT since it isn't working: 😉:

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Uchenna-ibute, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- GitHub flow is correctly used 👏

 ## Required Changes ♻️
-  Please Add a recorded video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in English. 
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code you find interesting or is proud of making.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️ ✔️ 

## Hi @Uchenna-ibute 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request ✔️ 
- work well organized ✔️ 
- great design✔️ 
- No linter errors✔️ 

- -  please always ensure your presentation is straight to the point and also say why you created the project or how important the piece of work will be to you or those around you. nice work

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**